### DISCUSSION TITLE
Replace Labs with Central HQ

### DISCUSSION THREAD

**Discussion by RealZLock** [voxeltycoon/issues#337] [General]
**Is your feature request related to a problem? Please describe.**
It is quite easy in the early/mid game to just throw laboratories near every resource and not have to transport anything to complete research. Not incentivized to build research hub.

**Describe the solution you'd like**
Replace laboratories with Central HQ and Regional HQs. When you start your company, you can place your one and only Central HQ, which is where you will complete research tasks. As you unlock more regions, you unlock more regional HQs where you can also complete research. You unlock your first regional HQ after you've opened your 4th total region. You get an additional regional HQ with every other region you unlock (or every third region perhaps). The roadmap includes politics, so the placement of HQs could then play a huge role in the political system when implemented just like in real life (i.e. companies are offered incentives to move or build an HQ in a region/city, companies with large HQs get influence in local government).

**Describe alternatives you've considered**
A simple limit on the number of labs you can build based on the number of regions unlocked could work as well, something like 2 labs at the start of the game and +1 lab for every region you unlock.

**Comment by victorleblais** [comment]
FYI: Company HQ is now a feature of the game (provides economical bonus to the city it's in), although labs are still a thing.
